src/content/app-platform/app-configuration/index.md

```diff
@@ -5,9 +5,10 @@ description: Documentation on the various levels of App configuration and how th
 menu:
   main:
     parent: app-platform
-weight: 20
+weight: 30
 aliases:
  - /reference/app-configuration/
+last_review_date: 2021-07-07
```

Comment:
Fixing the warning.

Reply:
FYI: A warning is not a blocker. Updating the `last_review_date` means: I checked the content and it is still up-to-date.

Reply:
Yes I know but we've worked on both pages recently and I've checked they are up to date. We just missed setting the date.